debian/control

```diff
@@ -2,22 +2,56 @@ Source: intelmq
 Maintainer: Sebastian Wagner <wagner@cert.at>
 Section: python
 Priority: optional
-Build-Depends: debhelper (>= 4.1.16), python3-all, python3-setuptools, quilt, dh-python, python-setuptools, dh-systemd, safe-rm, python3-requests, python3-redis, python3-dnspython, python3-psutil, python3-dateutil, python3-termstyle, python3-tz, lsb-release
+Build-Depends: debhelper-compat (= 13),
```

Comment:
With that requirement, we cannot build the package anymore on Ubuntu 16.04, 18.04, 20.04 and Debian 9, 10.

Reply:
I've changed that to `debhelper-compat (=10)` which should work for all of those, backports considered